src/verify.rs

```diff
@@ -5,25 +5,52 @@ use policy_fetcher::verify::Verifier;
 use std::io::prelude::*;
 use std::{collections::HashMap, fs::File};
 
+pub(crate) type VerificationAnnotations = HashMap<String, String>;
+
 pub(crate) async fn verify(
     url: &str,
-    docker_config: Option<DockerConfig>,
-    sources: Option<Sources>,
-    annotations: Option<HashMap<String, String>>,
+    docker_config: &Option<DockerConfig>,
+    sources: &Option<Sources>,
+    annotations: &Option<VerificationAnnotations>,
     key_file: &str,
+) -> Result<String> {
+    let verification_key = read_key_file(key_file)?;
+    let mut verifier = Verifier::new(sources.clone());
+    let verified_manifest_digest = verifier
+        .verify(
+            url,
+            docker_config.clone(),
+            annotations.clone(),
+            &verification_key,
+        )
+        .await?;
+
+    println!("Policy successfully verified");
```

Comment:
I think we should uses tracing.info for that.

Reply:
Done in https://github.com/kubewarden/kwctl/pull/84/commits/595796007840a6eb43ffd9bc95a004aebe3e98ed.